README.md

```diff
@@ -2,6 +2,15 @@
 
 ## Motivation
 
+## Installation
+ - For running TG code requirements_tg.txt can be used
```

Comment:
Conda environment based on `requirements_tg.txt` must be activated to install `plink-ng`.